test/functional/tests/stress/test_stress_change_io_class_config_io.py

```diff
@@ -0,0 +1,184 @@
+def test_stress_io_class_change_config_during_io_raw():
+    """
+    title: Set up IO class configuration from file during IO - stress.
+    description: |
+        Check Open CAS ability to change IO class configuration during running IO
+        on small cache and core devices.
+    pass_criteria:
+        - No system crash
+        - IO class configuration changes successfully
+        - No IO errors
+    """
+    cores_per_cache = 4
+
+    with TestRun.step("Prepare devices."):
+        cache_device = TestRun.disks['cache']
+        core_device = TestRun.disks['core']
+
+        cache_device.create_partitions([Size(150, Unit.MebiByte)])
+        core_device.create_partitions([Size(256, Unit.MebiByte)] * cores_per_cache)
+
+        cache_device = cache_device.partitions[0]
+        core_devices = core_device.partitions
+
+    with TestRun.step("Start cache in Write-Back mode and add core devices."):
+        cache = casadm.start_cache(cache_device, cache_mode=CacheMode.WB, force=True)
+        cores = []
+        for i in range(0, cores_per_cache):
+            cores.append(cache.add_core(core_devices[i]))
+
+    with TestRun.step("Create IO class configuration file for 33 IO classes with random allocation "
+                      "and priority value."):
+        generate_and_load_random_io_class_config(cache)
+
+    with TestRun.step("Run IO for all CAS devices."):
+        fio_task = start_async_func(run_io, cores, True)
+
+    with TestRun.step("In two-second time interval change IO class configuration "
+                      "(using random values in allowed range) and cache mode "
+                      "(between all supported). Check if Intel CAS configuration has changed."):
+        change_mode_thread = threading.Thread(target=change_cache_mode, args=[cache, fio_task])
+        change_io_class_thread = threading.Thread(target=change_io_class_config,
+                                                  args=[cache, fio_task])
+        change_mode_thread.start()
+        sleep(1)
+        change_io_class_thread.start()
+
+        while change_io_class_thread.is_alive() and change_mode_thread.is_alive():
+            sleep(10)
+
+        fio_result = fio_task.result()
+        if fio_result.exit_code != 0:
+            TestRun.fail("Fio ended with an error!")
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.asyncio
+async def test_stress_io_class_change_config_during_io_fs():
+    """
+    title: Set up IO class configuration from file for filesystems during IO - stress.
+    description: |
+        Check Intel CAS ability to change IO class configuration for different filesystems
+        during running IO on small cache and core devices.
+    pass_criteria:
+        - No system crash
+        - IO class configuration changes successfully
+        - No IO errors
+    """
+    cores_per_cache = 3
```

Comment:
Number of cores has to be equal to the number of tested filesystems so how about `cores_per_cache = len(list(Filesystem))`?